### PR TITLE
Move to settings.toml file

### PR DESCRIPTION
**Pull request by Samisafool**

### CODE REVIEW

**Comment by Samisafool** [comment]
Gonna add option to change stuff from GUI

**Comment by Samisafool** [comment]
Done :p

**Comment by Samisafool** [comment]
I think it's good to have a second eye, so exo can you review?

File: packwiz_gui.py [hunk 1]
```diff
@@ -9,61 +9,44 @@
...
-    else:
-        root = os.getcwd()
+        elif opt == "--reset-settings":
+            if os.path.isfile(f"{root}/settings.toml"):
+                os.remove(f"{root}/settings.toml")
+                recreatesettings(f"{root}/settings.toml")
+                print("Successfully reset settings.")
+                sys.exit()
+            else:
+                print("settings.toml does not exist! Run normally to create.")
```
Comment (ExoPlant) [inline packwiz_gui.py:58]: I don't like how it exits if you run --reset-settings when it's got no settings.toml

File: packwiz_gui.py [hunk 2]
```diff
@@ -80,20 +80,22 @@ def log(msg, logtype):
...
-            sys.exit()
-        sg.theme(settings["tktheme"])
-    elif settings["backend"] == "qt":
-        try:
-            import PySimpleGUIQt as sg
-        except ModuleNotFoundError:
-            print("You must install PySimpleGUIQt!")
-            sys.exit()
-        sg.theme(settings["qttheme"])
+    valid_backends = ["tk", "qt"]
```
Comment (ExoPlant) [inline packwiz_gui.py:99]: Suggestion: Add WK as valid backend

**Review by ExoPlant — CHANGES_REQUESTED** [review]

File: packwiz_gui.py (hunk 2 above)
Comment (Samisafool) [reply packwiz_gui.py:99]: wx*

and that will add extra weight to the actions artifact

File: packwiz_gui.py (hunk 1 above)
Comment (Samisafool) [reply packwiz_gui.py:58]: It exits but you can just start it again, and it's more consisted with exiting this way

Comment (Samisafool) [reply packwiz_gui.py:58]: consistent*

Comment (ExoPlant) [reply packwiz_gui.py:58]: I feel you should replace both the reset settings with `break` instead of `sys.exit()`

File: packwiz_gui.py (hunk 2 above)
Comment (ExoPlant) [reply packwiz_gui.py:99]: But WX looks better than tkinter by far

File: packwiz_gui.py (hunk 1 above)
Comment (Samisafool) [reply packwiz_gui.py:58]: ????
Break is for loops only

File: packwiz_gui.py (hunk 2 above)
Comment (Samisafool) [reply packwiz_gui.py:99]: It just looks more flat and is no different. I don't think?

Anyway, you can make tkinter look better. And you're already working on kivy

File: packwiz_gui.py (hunk 1 above)
Comment (Samisafool) [reply packwiz_gui.py]: Ok done

**Comment by Samisafool** [comment]
Okay, squash and merge time

**Pull request merged**